### DISCUSSION TITLE
How to downgrade back to 3.4.3

### DISCUSSION THREAD

**Discussion by monkemedia** [nuxt/nuxt#20928] [Questions] [answered]
So I have upgraded to 3.5.0 (npx nuxi upgrade --force) and seems to be causing issues. For some reason nested components are not being rendered. To make sure its the upgrade which is causing the issue, I need to go back to 3.4.3. The question is, how can I downgrade back to 3.4.3 ??

**Comment by warflash** [comment, marked as answer]
If you are using git then you may revert to the last version that worked. You can also remove the ^ that's infront of the nuxt version, downgrade to 3.4.3 in your package json, delete node_modules and packagelock and reinstall

**Comment by AloisSeckar** [comment]
I'd like to know too. Because I changed `nuxt` dependency back to `3.4.3`, did clean `npm install`, but still experiencing #20827 in VS Code. When I check `package.json` there are still `vue 3.3` dependencies comming from somewhere, even when I reverted all dependencies back to older state, where it worked.

My current dependencies are:

```
  "devDependencies": {
    "@nuxtjs/eslint-config-typescript": "12.0.0",
    "@playwright/test": "1.33.0",
    "@vitest/coverage-c8": "0.30.1",
    "eslint": "8.39.0",
    "nuxt": "3.4.3",
    "openapi-typescript": "5.4.1",
    "sass": "1.62.1",
    "vitest": "0.30.1"
  },
  "dependencies": {
    "@pinia/nuxt": "0.4.9",
    "consola": "2.15.3",
    "devextreme": "22.2.6",
    "devextreme-vue": "22.2.6",
    "pinia": "2.0.35"
  }
```

**Reply by maou-shonen** [reply]
Your problem should be fixed after deleting and regenerating node_modules and .nuxt,
If not, you can use [pnpm override](https://pnpm.io/package_json#pnpmoverrides).

**Reply by AloisSeckar** [reply]
I deleted everything possible (thats what I meant with "clean install")

I also tried 
```
"overrides" : {
  "vue" : "3.2.47"
}
```

but still no luck removing that silly bug :(

**Reply by maou-shonen** [reply]
You can use `pnpm why vue` to query the dependency tree, cover each one or adjust the version you need,
Or specify version 3.2 directly in your dependencies until your problem is fixed. (In terms of overriding the vue version I would use this😅because almost all packages will use peer to respect the version you choose )

**Reply by AloisSeckar** [reply]
yay, specifying `"vue" : "3.2.47"` in `devDependencies` helped...thank you
